arch/xtensa/src/esp32/esp32_start.c

```diff
@@ -146,6 +147,14 @@ void IRAM_ATTR __start(void)
   xtensa_early_serial_initialize();
 #endif
 
+#if defined(CONFIG_SPIRAM_BOOT_INIT)
+  esp_spiram_init_cache();
+  if (esp_spiram_init() != OK)
+    {
+      abort();
```

Comment:
I don't think `abort()` should be called here.

Reply:
Fixed! Thank you!